src/backend/metal/src/command.rs

```diff
@@ -14,416 +14,112 @@
 
 #![allow(missing_docs)]
 
-// use cocoa::foundation::NSRange;
+//use cocoa::foundation::NSRange;
 
-use core::{command, pso, shade, state, target, texture as tex};
+use core::{pso, shade, state, target, texture, command};
```

Comment:
command should probably go first since this list is in the alphabetical order, but that's minor